src/webgpu/listing_meta.json

```diff
@@ -898,6 +898,27 @@
   "webgpu:idl,constructable:gpu_errors:*": { "subcaseMS": 0.101 },
   "webgpu:idl,constructable:pipeline_errors:*": { "subcaseMS": 0.101 },
   "webgpu:idl,constructable:uncaptured_error_event:*": { "subcaseMS": 0.101 },
+  "webgpu:shader,execution,expression,binary,ai_arithmetic:addition:*": { "subcaseMS": 0 },
```

Comment:
note: with the latest version of `run-cts`, you can strip these out, run them again, and I'll put in real timing values!

Reply:
Done, gtk